README.md

```diff
@@ -1 +1,83 @@
-# kf-task-fhir-etl
\ No newline at end of file
+# Kids First FHIR ETL Task Service
+
+The Kids First FHIR ETL Task Service, built off of the [Kids First Data Ingest Library](https://github.com/kids-first/kf-lib-data-ingest), is a Python wrapper application, which
+
+- Extracts tables from the KF Dataservice DB;
+- Transforms the extracted tabular data to FHIR resources in JSON; and
+- Loads the transformed records into a Kids First FHIR Service.
+
+## Quickstart
+
+### Running ETL from Command Line Interface
+
+1. Make sure Python (>= 3.7) is installed.
+
+2. Obtain three sets of credentials as follows:
+
+   - Kids First Dataservice DB URL: Contact Kids First DRC DevOps Team.
+   - FHIR USERNAME and PASSWORD: The Kids First FHIR ETL uses basic authentication for POST, PUT, PATCH, and DELETE . Contact Kids First DRC DevOps Team.
+   - FHIR Cookie: Follow the instruction described [here](https://github.com/kids-first/kf-api-fhir-service).
+
+3. Clone this repository:
+
+```
+$ git clone https://github.com/kids-first/kf-task-fhir-etl.git
+$ cd kf-task-fhir-etl
+```
+
+4. Create and activate a virtual environment:
+
+```
+$ python3 -m venv venv
+$ source venv/bin/activate
+```
+
+5. Install dependencies:
+
+```
+(venv) $ pip install --upgrade pip && pip install -e .
+```
+
+6. Create a `.env` with the following environment variable names:
+
+```bash
+KF_DATASERVICE_DB_URL=<PUT-KF-DATASERVICE-DB-URL>
```

Comment:
Could you add a .env.example file with all of these env vars and set them equal to example values using localhost? Would be nice for anyone setting this up locally to get up and running more quickly